src/cljfx/coerce.clj

```diff
@@ -29,7 +29,8 @@
            [javafx.beans.value ObservableValue ChangeListener]
            [javafx.beans Observable InvalidationListener]
            [java.io InputStream]
-           [javafx.collections ListChangeListener]))
+           [javafx.collections ListChangeListener]
+           [javafx.animation Animation Interpolator PathTransition$OrientationType]))
```

Comment:
One error I made with `cljfx.coerce` is I introduced too many stuff here. Problem with this is `cljfx.coerce` is always reqiured, and all these classes are already loaded, which slows startup. Any app that does not use animation will still load animation classes. I'd like to prevent it from growth, that's why I started to move coercion function to `cljfx.fx.*` namespaces which are dynamically loaded, so user does not pay for not using them.

Reply:
Moved all animations out of coerce.